### PR TITLE
Fix mypy errors

### PR DESCRIPTION
**Pull request by MetRonnie**
This is a small change with no associated Issue.

### CODE REVIEW

File: cylc/uiserver/handlers.py [hunk 1]
```diff
@@ -166,13 +167,16 @@ class CylcAppHandler(JupyterHandler):
     3800ceaf9edf33a0171922b93ea3d94f87aa8d91/jupyterhub/
     singleuser/mixins.py#L826
     """
+    auth_level = None
```
Comment (datamel) [inline cylc/uiserver/handlers.py:170]: @oliver-sanders, is this `auth_level` still required?

[the remark this answers is not in the document]
Comment (oliver-sanders) [reply cylc/uiserver/handlers.py:170]: Doesn't look like it is:

```console
$ git grep auth_level
cylc/uiserver/handlers.py:    auth_level = None
````

[the remark this answers is not in the document]
Comment (MetRonnie) [reply cylc/uiserver/handlers.py]: I've removed it

**Review by oliver-sanders — APPROVED** [review]
(Tested the static handlers still work properly)

**Review by datamel — APPROVED** [review]

**Pull request merged**